.gitignore

```diff
@@ -1 +1,2 @@
-/doc/tags
+# /doc/tags
```

Comment:
Lets not comment out obsolete stuff, just delete it.

```suggestion
```

Reply:
that's comment as well, since not all people use 'tags', but just tags at current tag should had include 'doc' folder already
// but that's not a big deal, if you insisted, i can remove that/such 'comment'.